### ISSUE TITLE
Add support for `dotenvx decrypt` command to undo `dotenvx convert` command to a `.env` file

### ISSUE THREAD

**Issue by kwekewk** [dotenvx/dotenvx#247]
Currently we have
```
Usage: @dotenvx/dotenvx convert [options]
convert env file(s) to encrypted env file(s)
```
But not the opposite and 'dotenvx get' will give us json format, maybe something:
dotenvx get --format dotenv or dotenvx get --raw?

**Comment by motdotla** [comment]
great point. adding to the todos. this will probably come after a 1.0.0 release - at which time the commands `dotenvx encrypt` and `dotenvx decrypt` will be freed up (they've been moved to `dotenvx vault decrypt/encrypt`)

**Comment by motdotla** [comment]
effort started here: https://github.com/dotenvx/dotenvx/pull/294

will support by default rewriting the same file, but you can also send to stdout or a filename of your choice. stdout will let you pipe it to other commands to do what you want with it.

**Comment by motdotla** [comment]
this was released as part of 1.6.x - https://github.com/dotenvx/dotenvx/blob/main/CHANGELOG.md#160

at which time dotenvx was considered feature complete 🏁 https://dotenvx.com/blog/2024/07/11/feature-complete.html
